news/2022-07-25-mappers-guild-updates.md

```diff
@@ -0,0 +1,174 @@
+
+## Mapping contest tools
+
+How do you create a mapping contest? How do you anonymize beatmap submissions? How do you judge mapping contests without stone-age spreadsheets? Where does anyone even *find* mapping contests?
+
+Mapping contests have always faced systemic problems. The [**Mappers' Guild** contest listing](https://mappersguild.com/contests/listing) aims to solve them.
+
+![](/wiki/shared/news/2022-07-25-mappers-guild-updates/contest-listing.png)
+
+This system aims to act as a central hub for mapping contests.
+
+If you're a mapper, this helps you to...
+
+- Find all active mapping contests
+- Browse previous community mapping contests
+
+Honestly speaking, this system is much more useful for **contest creators**. If you're a contest host, this helps you to...
+
+- Make your contests more accessible (via the [contest listing](https://mappersguild.com/contests/listing) and the [osu!dev](https://discord.gg/ppy) Discord server's `#mappers-guild` channel)
+- Automatically [anonymize beatmap submissions](/wiki/shared/news/2022-07-25-mappers-guild-updates/contest-anon.png)
+- Find [users interested in screening/judging](/wiki/shared/news/2022-07-25-mappers-guild-updates/contest-helper.png)
+- Provide a platform for screening/judging staff (It's *much nicer* than spreadsheets. Seriously.)
+- Generate contest results for public display ([example](https://mappersguild.com/contests/results?contest=606267f26a56d154dccc55db))
+
+This system also offers [**Mappers' Guild** points](/wiki/Community/Mappers_Guild#rewards) for beatmaps that **exclusively** use Featured Artist content:
+
+- Hosting contest: 10 points
+- Submitting a valid entry: 5 points
+- Screening contest: 1 point
+- Judging contest: 1 point
+
+Words suck. Explore the [**Mappers' Guild** contest listing](https://mappersguild.com/contests/listing) for yourself and see if you're interested in long-overdue mapping contest organization!
+
+## Guest difficulty system changes
+
+A contentious part of the **Mappers' Guild** has always been its admission requirements. For those unfamiliar, you need **3 ranked maps** to create maps and earn rewards through the guild.
+
+This was silently changed recently!
+
+While you still need 3 ranked maps to host mapsets in the guild, **guest difficulties can come from any mappers**, regardless of how many ranked maps they have. Once a user reaches 3 ranked maps, any points they've earned from guest difficulties will be retroactively applied.
+
+This sounds like nonsense to anyone who's not familiar with the **Mappers' Guild** already, we know. In summary, the guild is now more accessible, and we just wanted to let you know!
+
+## Major contributors
+
+The guild has been *extremely active* since [our last homepage update](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild). **750+** new **Mappers' Guild** beatmaps have been ranked and **160+ Mappers' Guild** quests have been completed.
```

Comment:
```suggestion
The guild has been *extremely active* since [our last home page update](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild). **750+** new **Mappers' Guild** beatmaps have been ranked and **160+ Mappers' Guild** quests have been completed.
```